src/main/java/kitchenpos/order/eatinorder/domain/model/OrderTable.java

```diff
@@ -25,17 +26,57 @@ public class OrderTable {
     public OrderTable() {
     }
 
-    public OrderTable(UUID id, String name, int numberOfGuests, boolean occupied) {
+    public OrderTable(UUID id, OrderTableName name, int numberOfGuests, boolean occupied) {
+        validateNumberOfGuests(numberOfGuests);
         this.id = id;
         this.name = name;
         this.numberOfGuests = numberOfGuests;
         this.occupied = occupied;
     }
 
+    public OrderTable(UUID id, OrderTableName name) {
+        this(id, name, 0, false);
+    }
+
+    public OrderTable(UUID id, String name, int numberOfGuests, boolean occupied) {
+        this(id, new OrderTableName(name), numberOfGuests, occupied);
+    }
+
```

Comment:
VO객체로 온전히 값에 대한 책임을 수행하도록 만드는 것은 저도 수찬님과 같은 생각입니다. 😄 

그리고 이거는 제 개인적인 생각 및 취향(?) 이라서 그냥 가볍게 봐주시면 감사하겠습니다. 😅 

생성자가 필요한 경우는 새로운 데이터 생성 및 검증, 조회된 데이터 생성 이라고 생각합니다. 
처음에 OrderTable을 생성할때 필요한 필드는 이름입니다. 
그래서 이름만 필드값으로 하고 id는 엔티티에서 생성하는 방식으로 하시면 어떨까 생각합니다. 

```suggestion
    public OrderTable(string name) {
        this(UUID.randomUUID(), name, 0, false);
    }

    public OrderTable(UUID id, String name, int numberOfGuests, boolean occupied) {
        validateNumberOfGuests(numberOfGuests);
        this.id = id;
        this.name = new OrderTableName(name);
        this.numberOfGuests = numberOfGuests;
        this.occupied = occupied;
    }

```

Reply:
UUID는 entity가 사실 도메인 엔티티 + db 엔티티이기 때문에 id를 생성자 시점에서 할당해도 괜찮을것 같은데 리뷰어님 생각은 어떠신가요?
(ORM 사용 이유도 테이블과 객체 매핑이기 때문에 jpa entity이므로 객체를 생성한 후에는 그 객체는 항상 영속화된다고 생각해서 이렇게 적용해봤습니다!)